soh/macosx/soh.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+export DATA_SHARE="$HOME/Library/Application Support/com.shipofharkinian.soh"
+export ASSETDIR="$(mktemp -d /tmp/assets-XXXXX)"
+export SNAME=$(dirname "$0")
+export NAME=${SNAME%/soh.app*}
+export LIBPATH=${SNAME%/MacOS*}
+
+while [ ! -e "$DATA_SHARE/oot.otr" ]; do
+	cp -r "$SNAME/assets" "$ASSETDIR"
+	mkdir -p "$ASSETDIR"/tmp
+	mkdir -p "$ASSETDIR"/Extract/assets
+	DROPROM=$(osascript -ss - "$ASSETDIR" <<EOF
+	set romFile to choose file of type {"b64","n64","v64","z64"} with prompt "Please select your ROM:"
+	set destinationFolder to POSIX file "$ASSETDIR"
+	tell application "Finder"
+	duplicate romFile to destinationFolder
+	end tell
+	EOF
+	)
+	$DROPROM
+	if [ ! -e "$ASSETDIR"/*.*64 ]; then
+		echo "no ROM"
+		osascript -e 'display dialog "Select ROM to generate OTR" giving up after 5'
+		rm -r "$ASSETDIR"
+		exit
+	fi   
+	cp "$ASSETDIR"/*.*64 "$ASSETDIR"/tmp/rom.z64
+	cp -r "$ASSETDIR"/assets/game/ship_of_harkinian "$ASSETDIR"/Extract/assets/
+	sleep 3
+	cd "$ASSETDIR"
+	export ROMHASH="$(shasum "$ASSETDIR"/tmp/rom.z64 | awk '{ print $1 }')"
+	case "$ROMHASH" in
+	cee6bc3c2a634b41728f2af8da54d9bf8cc14099)
+  		export ROM=GC_NMQ_D;;
+	0227d7c0074f2d0ac935631990da8ec5914597b4)
+		export ROM=GC_NMQ_PAL_F;;
+	*)
+		WRONGHASH=$(osascript -ss - "$ROMHASH" <<EOF
+		display dialog "Incompatible ROM hash $ROMHASH" \
+		with title "Incompatible ROM hash" \
+		with icon caution \
+		giving up after 5
+		EOF
+		)
+		$WRONGHASH
+		rm -r "$ASSETDIR"
+		exit;;
+	esac
+	echo $ROM
+
+	osascript -e 'display notification "Processing OTR..." with title "SOH: Generating OTR"'
+	mkdir -p /usr/local/opt/glew/lib/
+	ln -s "$LIBPATH"/libs/libGLEW.2.2.0.dylib /usr/local/opt/glew/lib/libGLEW.2.2.dylib
```

Comment:
`install_name_tool` instead of this. this is requiring homebrew, don't require users to use homebrew

Reply:
Corrected to the rpath of the CI  `/opt/local/lib/`
`install_name_tool` will fail, because ZAPD needs the absoule path to the libs in the bundle, and the bundle can be located in any location.

Reply:
rpath can be relative to the binary. this is broken for people who don't have /usr/local/opt/glew/lib folder.

Reply:
(for example, i do not have this folder)

Reply:
and now it's broken for people who don't have /opt/local/lib?

Reply:
It is only looking at the rpath of ZAPD and placing a symlink where it expects to find it from the bundle. soh will always launch from the same relative position to its libs, but ZAPD creates a temp directory to do all of its work, and then cleans up after the assets are extracted from the rom. The temp directory changes on each launch, therefore breaking any relative paths.